admin-dev/themes/new-theme/scss/components/_toggable_field.scss

```diff
@@ -50,6 +50,14 @@
   }
 }
 
+#attributes-generator {
+  .tokenfield {
+    .token-input {
+      width: 100% !important;
```

Comment:
Sometime using `!important` can still be acceptable in particular cases, that could be one but:

It will make the input in a new line after the first tag is added, is this something we want?

If yes, then you'll have to ignore stylelint there by using `// stylelint-disable-next-line`

Reply:
This rule is used only in the combination tab of a the edit product page and it fit perfectly the needed.

Thanks for the ignore stylelint rule. I'll add `/* stylelint-disable-next-line declaration-no-important */` . I think it's less permissive